### PR TITLE
Edit for clarity

### PR DESCRIPTION
**Pull request by wiggitywhitney**

### CODE REVIEW

File: manuscript/develop/devspace-helm.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ devspace dev --namespace dev
 
 go run .
 
-# Check the newly opened browser tab
+# Open localhost:8080 in a browser
```
Comment (vfarcic) [inline manuscript/develop/devspace-helm.md:28]: It should open a new browser tab automatically. Was that not the case?

File: manuscript/develop/devspace-carvel.md [hunk 2]
```diff
@@ -52,8 +52,7 @@ go run .
 # In the first terminal
 exit
 
-# In the second terminal
-exit
+# Close the second terminal
```
Comment (vfarcic) [inline manuscript/develop/devspace-carvel.md:55]: Shouldn't `exit` close the terminal?

File: manuscript/develop/devspace-helm.md [hunk 3]
```diff
@@ -46,8 +46,7 @@ go run .
 # In the first terminal
 exit
 
-# In the second terminal
-exit
+# Close the second terminal
```
Comment (vfarcic) [inline manuscript/develop/devspace-helm.md:49]: Shouldn't `exit` close the terminal?

File: manuscript/develop/devspace-kustomize.md [hunk 4]
```diff
@@ -46,8 +46,7 @@ go run .
 # In the first terminal
 exit
 
-# In the second terminal
-exit
+# Close the second terminal
```
Comment (vfarcic) [inline manuscript/develop/devspace-kustomize.md:49]: Shouldn't `exit` close the terminal?

File: manuscript/develop/devspace-helm.md (hunk 1 above)
Comment (wiggitywhitney) [reply manuscript/develop/devspace-helm.md:28]: I didn't mean to change that, sorry.  Changing it back

**Pull request merged**